src/nupic/algorithms/TemporalMemory.cpp

```diff
@@ -283,7 +283,7 @@ static void growSynapses(Connections &connections,
   // Check if we're going to surpass the maximum number of synapses. //TODO delegate this to createSynapse(segment)
   const size_t overrun = (connections.numSynapses(segment) + nActual - maxSynapsesPerSegment);
   if (overrun > 0) {
-    destroyMinPermanenceSynapses(connections, rng, segment, overrun, prevWinnerCells);
+    destroyMinPermanenceSynapses(connections, rng, segment, static_cast<Int32>(overrun), prevWinnerCells);
```

Comment:
let's make this match, let overrun be size_t, and match nDestroy above

Reply:
> let's make this match, let overrun be size_t, and match nDestroy above

Ok, I will change the calling arguments in destroyMinPermananceSynapses( ) and change nDestroy and the for loop i argument to be size_t.